pyiron/sphinx/base.py

```diff
@@ -1242,6 +1243,38 @@ def write_input(self):
             f.write("include <parameters.sx>;\n\n")
             f.write(self.input.sphinx.to_sphinx(indent=0))
 
+    def get_charge_density(self):
+        """
+        Gets the charge density from the hdf5 file. This value is normalized by the volume
+
+        Returns:
+            atomistics.volumetric.generic.VolumetricData instance
+        """
+        if not self.status.finished:
```

Comment:
In some cases, the status could be `warning` or `not_converged` and still have the charge density stored. So I think we should have something like, `if self.status in ["finished", "warning", "not_converged"]`. This has to be done for VASP too.